style.scss

```diff
@@ -1,58 +1,6 @@
-$darkGreen: #21594e;
-$green: #3db14c;
-$limeGreen: #d9ed14;
-$yellow: #fef97a;
-$red: #a31423;
+@import "Imports/variables";
```

Comment:
Much better!

I'd actually suggest tidying this up a tiny bit, my suggestion is to make two new folders
1) scss folder, containing this file and the imports folder
2) CSS folder, containing the output css file

Just because a folder called Imports could be a little bit confusing!